src/script/interpreter.cpp

```diff
@@ -1297,9 +1297,12 @@ void PrecomputedTransactionData::Init(const T& txTo)
 
     // Cache is calculated only for transactions with witness
     if (txTo.HasWitness()) {
-        hashPrevouts = GetPrevoutHash(txTo);
-        hashSequence = GetSequenceHash(txTo);
-        hashOutputs = GetOutputsHash(txTo);
+        hashPrevouts = GetPrevoutSHA256(txTo);
+        hashSequence = GetSequenceSHA256(txTo);
+        hashOutputs = GetOutputsSHA256(txTo);
+        SHA256Uint256(hashPrevouts);
```

Comment:
Would prefer to do this as a one-liner: `hashPrevouts = SHA256Uint256(GetPrevoutSHA256(txTo))`, as you do below.

Reply:
The point of not being a one liner is to make it easier for future work to just add a line of copying.